src/components/Envelope.vue

```diff
@@ -19,9 +19,11 @@
 				v-if="mailbox.isUnified && hasMultipleAccounts"
 				class="mail-message-account-color"
 				:style="{'background-color': accountColor}" />
-			<div
+			<Star
 				v-if="data.flags.flagged"
-				class="app-content-list-item-star icon-starred"
+				fill-color="#f9cf3d"
```

Comment:
https://github.com/nextcloud/mail/pull/6633 let's do without

Reply:
then should we change also the important icon? This is also over the avatar so the theme doesnt influence the visibility

Reply:
if only the color is used to distinguish between the two states then yes

Reply:
not really, on the avatar its only "orange", the other state is removing the favorite or important icon. 
On the menu it changed color when it was not favorite. 

So, you're proposing these 2 icons to be dark grey now?
![favoriteimportant](https://user-images.githubusercontent.com/12728974/175949704-6bde9be0-7f8b-4eb6-bf5d-c442247d9336.png)

Reply:
I'm sorry, I thought the PR touched the envelope menu. The overlay icon should probably keep its color :+1: